packages/interactor/src/match.ts

```diff
@@ -26,6 +26,14 @@ export class Match<E extends Element, S extends InteractorSpecification<E>> {
   get sortWeight(): number {
     return this.matchLocator.sortWeight + this.matchFilter.sortWeight;
   }
+
+  elementDescription(): string {
+    let tag = this.element.tagName.toLowerCase();
+    let attrs = Array.from(this.element.attributes).map((attr) => {
+      return `${attr.name}="${attr.value}"`
```

Comment:
@jnicklas Just occured to me. Do we need to HTMLEncode the attribute value here? It's purely for debug purposes, but it would look weird if the `.value` had a `"` character for example

Reply:
Nice catch!